This patch is only applicable to version 1.1.11 of KDEpy, where the cutils.pyx file is
missing and the distribution already inlcudes the cutils.c file.
Author: Stamen Miroslavov <stamen.miroslavov@bsc.es>
Barcelona Supercomputing Center 2026
diff -ruNBbw a/setup.py b/setup.py
--- a/setup.py	2024-10-29 12:54:03.000000000 +0100
+++ b/setup.py	2026-09-29 16:40:04.831998318 +0200
@@ -16,7 +16,7 @@
     packages=["KDEpy"],
     cmdclass={"build_ext": build_ext},
     include_dirs=[np.get_include()],
-    ext_modules=cythonize(
-        [Extension("KDEpy._cutils", ["KDEpy/cutils_ext/cutils.pyx"])],
-    ),
+    ext_modules=[
+        Extension("KDEpy._cutils", ["KDEpy/cutils_ext/cutils.c"]),
+    ],
 )
